app/models/user.rb

```diff
@@ -121,6 +124,7 @@ def send_confirmation_instructions
     return unless active?
 
     super
+    update!(initial_confirmation_sent: true)
```

Comment:
not certain this is the best way of showing a different form when the user has already been invited before, happy to take suggestions if you can think of a better way!